packages/app-desktop/gui/NoteEditor/utils/useFormNote.test.ts

```diff
@@ -82,6 +83,37 @@ describe('useFormNote', () => {
 		formNote.unmount();
 	});
 
+
+	it('should preserve value of is_conflict on save', async () => {
+		const testNote = await Note.save({ title: 'Test Note!', is_conflict: 1 });
+
+		const makeFormNoteProps = (): HookDependencies => {
+			return {
+				...defaultFormNoteProps,
+				noteId: testNote.id,
+			};
+		};
+
+		const formNote = renderHook(props => useFormNote(props), {
+			initialProps: makeFormNoteProps(),
+		});
+		await formNote.waitFor(() => {
+			expect(formNote.result.current.formNote).toMatchObject({
+				is_conflict: 1,
+				title: testNote.title,
+			});
+		});
+
+		// Should preserve is_conflict after save.
+		expect(await formNoteToNote(formNote.result.current.formNote)).toMatchObject({
+			is_conflict: 1,
+			deleted_time: 0,
+			title: testNote.title,
+		});
+
+		formNote.unmount();
+	});
```

Comment:
I'm not sure how useful this automated test is &mdash; the bug is related to the interaction of `useFormNote` with `reducer.ts`. As such, an integrated/end-to-end test would make more sense. However, creating conflict notes in Playwright end-to-end tests is more complicated than creating conflicts in Jest tests.